skymp5-server/cpp/addon/main.cc

```diff
@@ -395,7 +395,7 @@ ScampServer::ScampServer(const Napi::CallbackInfo& info)
     auto reloot = serverSettings["reloot"];
     for (auto it = reloot.begin(); it != reloot.end(); ++it) {
       std::string recordType = it.key();
-      auto timeMs = static_cast<int>(it.value());
+      auto timeMs = static_cast<uint64_t>(it.value());
```

Comment:
596.5 hours before actually sounded like something that is ok :thinking: 

But should be 64-bit, yeah